### ISSUE TITLE
Question: Example of configuration tslint-jsdoc-rules

### ISSUE THREAD

**Issue by petrovic-ivan** [zorkow/javascript-environments#3]
Do you have some example how to use `tslint-jsdoc-rules`? After I install how I have to configure to see errors on class members which are not documented?

**Comment by SuperCorks** [comment]
Doesn't seem to work with me either...

here's the head of my **tslint.json**:

```
{
  "rulesDirectory": [
    "node_modules/codelyzer",
    "node_modules/tslint-jsdoc-rules/lib"
  ],
  "rules": {
    "jsdoc-require": [
      true,
      "no-constructors"
    ],
```

Yet I see no errors on methods that don't respect the rule...

**Comment by ArielGueta** [comment]
Hey, did you found the solution?

**Comment by SuperCorks** [comment]
Actually it does work, but it won't work with your IDE's integrated TSLINT validator. I can't seem to recall where I saw why it doesn't work, but the main idea was that it was validated after some initial typescript compilation steps and your IDE's validator doesn't go that deep.

I think it had to do with this:
https://palantir.github.io/tslint/usage/type-checking/

But essentially, when I ran the cli's lint command, I saw errors.

Btw, there are native rules for doc checking.
https://palantir.github.io/tslint/rules/completed-docs/

Check the comments in:
https://github.com/palantir/tslint/issues/2767
